### DISCUSSION TITLE
Line numbers

### DISCUSSION THREAD

**Discussion by cooper-david-a** [mgreminger/EngineeringPaper.xyz#293] [Ideas]
Line/Cell numbers would be helpful for longer sheets.

**Comment by mgreminger** [comment]
Thanks for the suggestion. Are you thinking more equation numbers (numbers for math cells only) or numbers for all cells, including documentation cells? I'd probably lean towards equation numbers, would give the user something to refer to in documentation cells.

**Comment by cooper-david-a** [comment]
The only problem I was actually having was that, after inserting a sheet into a similar one, I would lose track of the insertion point.
Now that I am typing that out, it occurs to me that I could have done a documentation cell to mark my place before hand.

However, equation numbers do seem like a good idea either way.

**Comment by mgreminger** [comment]
Thanks for the clarification, I see how that could be an issue when inserting long sheets. The best fix for this might be for the first cell of the inserted sheet to be selected and scrolled into view after the insertion.

**Comment by mgreminger** [comment]
I just rolled out an update that selects the first inserted cell when inserting an existing sheet (it will get scrolled into view, if necessary). This should make it clear to the user where the insertion starts. Thanks for raising the issue, much appreciated!
